hcipy/aperture/realistic.py

```diff
@@ -155,6 +155,51 @@ def func(grid):
 		return obstructed_aperture(grid) * spider1(grid) * spider2(grid) * spider3(grid) * spider4(grid)
 	return func
 
+def make_hale_aperture(normalized=False, with_spiders=True):
+	'''Make the Hale Telescope aperture.
+
+	The size of the spiders and the attachements were based on the actual pupil measurements from Figure 2 of [Galicher2019]_
+
+	.. [Galicher2019] Raphael Galicher et al., "Minimization of non-common path aberrations at the Palomar telescope using a self-coherent camera." Astronomy & Astrophysics 631 (2019): A143.
+	
+	Parameters
+	----------
+	normalized : boolean
+		If this is True, the outer diameter will be scaled to 1. Otherwise, the
+		diameter of the pupil will be 5.08 meters.
+	with_spiders: boolean
+		If this is False, the spiders will be left out.
+
+	Returns
+	-------
+	Field generator
+		The Hale Telescope aperture.
+	'''
+	pupil_diameter = 5.08 # meter
+	central_obscuration_diameter = 1.86 # meter
+	spider_width = 2 * 0.024 # meter
+	central_obscuration_ratio = central_obscuration_diameter / pupil_diameter
+
+	if normalized:
+		spider_width /= pupil_diameter
+		pupil_diameter = 1.0
```

Comment:
You're not normalizing central obscuration diameter.

Reply:
That's not necessary because I already calculate the central obscuration ratio a couple lines before this one.

Reply:
I always made the `if normalized` a kind of barrier, where no non-normalized values can pass through, even for values that are not used later on. That makes it harder to make normalization errors upon refactoring. But this'll do for now.

FYI: don't click resolved if you're not certain a comment is actually addressed. It's supposed to be the reviewer that checks and decides if all their comments are addressed sufficiently and click resolve if that is the case. It's not a checklist for you.